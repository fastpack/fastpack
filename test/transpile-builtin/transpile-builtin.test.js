const { fpack } = require("../helpers.js");

process.chdir(__dirname);

test("transpile flow inside jsx", async () => {
  expect(
    (await fpack(["index-1.js", "--preprocess=^.*"])).bundle
  ).toMatchSnapshot();
});
