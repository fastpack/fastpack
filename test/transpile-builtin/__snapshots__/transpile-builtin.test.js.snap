// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transpile flow inside jsx 1`] = `
"(function() {

/* index$$_$$1 */

// @flow
// The following test makes sure that builtin transpiler strips type annotations
// from inside JSX
const $i__index$$_$$1__Component = props => React.createElement(\\"div\\", null, item => {
    
  }
  );

const $n__index$$_$$1 = { exports: {} };
})()
"
`;
